sdk/core/azure-core/azure/core/messaging.py

```diff
@@ -69,35 +69,30 @@ class CloudEvent(object):  # pylint:disable=too-many-instance-attributes
     :ivar extensions: A CloudEvent MAY include any number of additional context attributes
      with distinct names represented as key - value pairs. Each extension must be alphanumeric, lower cased
      and must not exceed the length of 20 characters.
-    :vartype extensions: Dict
+    :vartype extensions: dict
     """
 
-    def __init__(self, source, type, **kwargs):  # pylint: disable=redefined-builtin
-        # type: (str, str, **Any) -> None
-        self.source = source  # type: str
-        self.type = type  # type: str
-        self.specversion = kwargs.pop("specversion", "1.0")  # type: Optional[str]
-        self.id = kwargs.pop("id", str(uuid.uuid4()))  # type: Optional[str]
-        self.time = kwargs.pop("time", datetime.now(TZ_UTC))  # type: Optional[datetime]
-
-        self.datacontenttype = kwargs.pop(
-            "datacontenttype", None
-        )  # type: Optional[str]
-        self.dataschema = kwargs.pop("dataschema", None)  # type: Optional[str]
-        self.subject = kwargs.pop("subject", None)  # type: Optional[str]
-        self.data = kwargs.pop("data", None)  # type: Optional[object]
-
-        try:
-            self.extensions = kwargs.pop("extensions")  # type: Optional[Dict]
-            for (
-                key
-            ) in self.extensions.keys():  # type:ignore # extensions won't be None here
+    def __init__(
+        self, source: str, type: str, **kwargs: Any  # pylint: disable=redefined-builtin
```

Comment:
Do we want to use something like 
```py
def __init__(self, source: str, type: str, *, specversion,  **kwargs)
```
for those key word only arguments?

Reply:
Never mind.

We can do that in a separate PR if we want.

The PR is big enough that we don't want to make it more complicated.

Reply:
Changed them all to kwargs